spec/lib/runner_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "active_support/testing/time_helpers"
+
+RSpec.describe Runner do
+  include ActiveSupport::Testing::TimeHelpers
+
+  describe ".call" do
+    context "after 10am" do
```

Comment:
This isn't actually yesterday it's just what time you are executing this class. So it will execute fail if this job is executed before 10am and succeed if not.

Reply:
Oh yeah, I got them the wrong way round. Looks better now, anyway.